.env

```diff
@@ -0,0 +1,9 @@
+# Environment variables declared in this file are automatically made available to Prisma.
+# See the documentation for more detail: https://pris.ly/d/prisma-schema#accessing-environment-variables-from-the-schema
+
+# Prisma supports the native connection string format for PostgreSQL, MySQL, SQLite, SQL Server, MongoDB and CockroachDB.
+# See the documentation for all the connection string options: https://pris.ly/d/connection-strings
+
+DATABASE_URL="postgres://justin@localhost:5432/postgres"
+SECRET="SECRET"
```

Comment:
This should be included in the `gitignore`

Reply:
.env is in .gitignore, but forgot to remove it and recommit.